### DISCUSSION TITLE
创建知识库时候 提示 modeldata 不存在

### DISCUSSION THREAD

**Discussion by 157677678** [labring/sealos#4301] [Q&A]
新部署的 私有云 部署后 创建知识库 提示  modeldata 不存在
![image](https://github.com/labring/sealos/assets/67778969/498623ca-c13d-4b0b-8169-4285499cfb6e)
公网的没有这个问题 公网的有什么新变化吗？

**Comment by bxy4543** [comment]
请提交至 fastgpt
